src/borg/key.py

```diff
@@ -171,7 +171,7 @@ def id_hash(self, data):
 
     def encrypt(self, chunk):
         chunk = self.compress(chunk)
-        self.nonce_manager.ensure_reservation(num_aes_blocks(len(chunk.data)))
+        self.nonce_manager.ensure_reservation(self.cipher.block_count(len(chunk.data)))
```

Comment:
The self.cipher.block_count method is a precise thing, it does the same thing correctly for any cipher in use that num_aes_blocks did correct for AES.

note that the global block_count *function* can be (and is) used at some other places in a more sloppy way, when the cipher block size is not known and we just assume it is not less than 16B.

And there is no future code yet, so this is just making things work correctly for the present code.